File: app/src/main/java/ru/magflayer/spectrum/presentation/common/listeners/SimpleAnimatorListener.java
package ru.magflayer.spectrum.presentation.common.listeners;

import android.animation.Animator;


public abstract class SimpleAnimatorListener implements Animator.AnimatorListener {

    @Override
    public void onAnimationStart(Animator animation) {
        //do nothing
    }

    @Override
    public void onAnimationEnd(Animator animation) {
        //do nothing
    }

    @Override
    public void onAnimationCancel(Animator animation) {
        //do nothing
    }

    @Override
    public void onAnimationRepeat(Animator animation) {
        //do nothing
    }
}

File: app/src/main/java/ru/magflayer/spectrum/presentation/common/listeners/SimpleAnimatorListener.kt
package ru.magflayer.spectrum.presentation.common.listeners

import android.animation.Animator

abstract class SimpleAnimatorListener : Animator.AnimatorListener {

    override fun onAnimationStart(animation: Animator) {
        // do nothing
    }

    override fun onAnimationEnd(animation: Animator) {
        // do nothing
    }

    override fun onAnimationCancel(animation: Animator) {
        // do nothing
    }

    override fun onAnimationRepeat(animation: Animator) {
        // do nothing
    }
}
